resources/usa_death/client.lua

```diff
@@ -93,6 +93,13 @@ function respawnPed(ped,coords)
 	TriggerEvent("chatMessage", "", { 0, 0, 0 }, "^1^*[RESPAWN] ^r^7You wake up at the local hospital, and can't seem to remember events leading up to now...")
 end
 
+RegisterNetEvent("usa_death:epi")
+AddEventHandler("usa_death:epi", function(bool)
+	if IsEntityDead(GetPlayerPed(-1)) then
```

Comment:
```suggestion
	if IsEntityDead(PlayerPedId()) then
```
I think that is a cleaner way to get the cur ped handle